patches/bess/0012-Dpdk-concurrent-hash-support-pipeline-improvement.patch

```diff
@@ -463,15 +501,21 @@ index 85deeafc..b6b2f728 100644
  
  class WildcardMatch final : public Module {
   public:
-@@ -144,6 +151,7 @@ class WildcardMatch final : public Module {
+@@ -144,10 +151,13 @@ class WildcardMatch final : public Module {
          values_(),
          tuples_() {
      max_allowed_workers_ = Worker::kMaxWorkers;
 +    { tuples_.resize(MAX_TUPLES); }
    }
  
    CommandResponse Init(const bess::pb::WildcardMatchArg &arg);
-@@ -164,13 +172,26 @@ class WildcardMatch final : public Module {
+ 
++  void DeInit() override;
++
```

Comment:
Do Deinit is ok for other cleanup scenarios as well?? for example allocating object on heap and then deallocating it.